--- mobile-broadband-provider-info-20100122/serviceproviders.xml	2010-01-22 22:20:34.000000000 +0100
+++ mobile-broadband-provider-info-20100122/serviceproviders.xml.foo	2013-06-14 08:56:40.376905806 +0200
@@ -1733,13 +1733,8 @@ conceived.
 		<gsm>
 			<network-id mcc="244" mnc="91"/>
 
-			<apn value="internet.saunalahti">
-				<name>Postpaid (contract)</name>
-				<dns>192.89.123.230</dns>
-				<dns>192.89.123.231</dns>
-			</apn>
 			<apn value="internet">
-				<name>Prepaid (no contract)</name>
+				<name>Postpaid (contract)</name>
 				<dns>192.89.123.230</dns>
 				<dns>192.89.123.231</dns>
 			</apn>
